bin/improver-ecc

```diff
@@ -135,16 +135,13 @@ def main():
     # Safe to now actually do the work...
     cube = iris.load_cube(args.input_filepath)
 
-    if not cube.coords('percentile_over_realization'):
-        msg = ('The input cube: {} does not have a '
-               'percentile_over_realization dimension.'.format(cube))
-        raise CoordinateNotFoundError(msg)
+    perc_coord = find_percentile_coordinate(cube)
```

Comment:
I'm curious as to why you put this here rather than in the plugin. It doesn't matter too much, I was just wondering.

Reply:
I was repeating what originally happened in improver-ecc (i.e. test for percentile coord before running the plugin) but it would probably simplify things if I did move this into the plugin. This would be a bit more of a change to the functionality of the plugin but I don't think it is called from anywhere but this CLI.

Reply:
Now moved to plugin.